packages/cron/README.md

```diff
@@ -1,73 +1,103 @@
 
 ```ts
+// Main bot file
+// Be sure to register the plugin before instantiating the client.
 import '@kingsworld/plugin-cron/register';
 ```
 
-If you want to set the default cron job timezone for all your cron jobs, you can do so within the client options. A list of TZ identifiers can be found on [Wikipedia](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
+Then, you can configure the plugin in the configuration options in your SapphireClient extension class or initializer. This will either be located in your new SapphireClient constructor call, or super in your constructor method if you use an extension class.
 
 ```ts
-new SapphireClient({
-	...otherClientOptions,
+const options = {
+	...otherClientOptionsGoHere,
 	cron: {
-		// the cron object is optional
-		defaultTimezone: 'Europe/London' // the cron package defaults to UTC
+		/**
+		 * Whether to disable Sentry cron monitoring
+		 * @default true
+		 */
+		disableSentry: false,
+		/**
+		 * The timezone to use for the cron tasks
+		 * @default 'UTC'
+		 */
+		timeZone: 'Europe/London' // set the timezone
 	}
-});
+};
 ```
 
-However, if you would like to do so for a single task, you can do so in the cron task options
+-   The `disableSentry` option is used to disable [Sentry's cron monitoring](https://docs.sentry.io/product/crons/). If you do not use Sentry, you can safely ignore this option. If you use Sentry, you can set this to `false` to enable cron monitoring.
 
-```js
-{
-    cronTime: "* * * * *", // every minute
-    timeZone: "Europe/London"
-}
-```
+In order to use the cron tasks anywhere other than a piece (commands, arguments, preconditions, etc.), you must first import the `container` property of `@sapphire/framework`. For pieces, you can simply use `this.container.cron` to access this plugin's methods.
 
-### Creating a task
+This is a simple example of how to start a task from a service.
 
-Cron Tasks come with their own Sapphire store. They must be located within a `cron-tasks` directory alongside your other stores.
+```typescript
+import { container } from '@sapphire/framework';
 
-```
-src
-├── listeners
-|   └── ready.ts
-└── cron-tasks
-    └── ping.ts
+export class MyAwesomeService {
+	public createAwesomeTask() {
+		const task = container.cron.store.get('my-awesome-task');
+
+		task.job.start();
+	}
+}
 ```
 
-Using decorators from [@sapphire/decorators](https://www.npmjs.com/package/@sapphire/decorators):
+This is a simple example of how to start all tasks from a service.
 
-```ts
-import { ApplyOptions } from '@sapphire/decorators';
-import { CronTask } from '@kingsworld/plugin-cron';
+```typescript
+import { container } from '@sapphire/framework';
 
-@ApplyOptions<CronTask.Options>({
-	cronTime: '* * * * *'
-})
-export class PingPong extends CronTask {
-	run() {
-		this.container.logger.info('Ping Pong! 🏓');
+export class MyAwesomeService {
+	public createAwesomeTask() {
+		await container.cron.startAll();
```

Comment:
It might be worth noting that the `startAll` method is called automatically when the client is ready. Also, the method is synchronous, so the await is not needed.

https://github.com/Kings-World/sapphire-plugins/blob/496b6be8f7b533b8b4fbba7fde5b58211b2ae776/packages/cron/src/register.ts#L21-L23

https://github.com/Kings-World/sapphire-plugins/blob/496b6be8f7b533b8b4fbba7fde5b58211b2ae776/packages/cron/src/lib/structures/CronTaskStore.ts#L10-L18